content/projects/microbial/index.md

```diff
@@ -0,0 +1,137 @@
+---
+description: "Whether you are analysing microbiome samples or bacterial isolates, long reads or short, shotgun or 16S, genomics transcriptomics or proteomics, this is the place to be!"
+autotoc: true
+title: "Microbial data analysis in Galaxy"
+---
+
+Whether you are analysing microbiome samples or bacterial isolates, long reads or short, shotgun or 16S, genomics transcriptomics or proteomics, **this is the place to be**!
+
+**[microGalaxy](#microgalaxy-community)** is a community of practice in the Galaxy community for anything regarding microbial data analysis in Galaxy!
+
+
+# What is available?
+
+
+## Tools
+
+**Many tools for microbial data analysis are freely available** in the ToolShed which can be installed on any Galaxy server. To keep an overview, the *[microGalaxy](#microgalaxy-community)* community of practice maintains a [**curated list**](https://docs.google.com/spreadsheets/d/1Nq_g-CPc8t_eC4M1NAS9XFJDflA7yE3b9hfSg3zu9L4/edit#gid=101894501) of the **150+ tools** available.
+
+If tools are missing or information is not up-to-date in the list, please help us! Contact Saskia or Bérénice about it.
+
+
+## Workflows and tutorials
+
+Several curated Galaxy workflows are publicly available for different kinds of microbial data analysis. Many of these are accompanied by comprehensive [GTN Tutorials](https://training.galaxyproject.org) that will guide you through the analysis step by step.
+
+*The lists below are far from being exhaustive and up-to-date. Something missing? Feel free to update the page or reach to us*
+
+### Microbiome
+
+| Target | Technique | Sequencing | Analysis | GTN Tutorial | Workflow | Notes |
+|--------|-----------|------------|----------|--------------|----------|-------|
+| Bacteria | 16S | Short reads | Taxonomic Profiling | <a href="https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/mothur-miseq-sop/tutorial.html" class="fas fa-graduation-cap" aria-hidden="true"></a> | <a href="https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/mothur-miseq-sop/workflows/" class="fas fa-share-alt" aria-hidden="true"></a> | mothur SOP
+| Bacteria | 16S | Long reads | Taxonomic Profiling  | <a href="https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/nanopore-16S-metagenomics/tutorial.html" class="fas fa-graduation-cap" aria-hidden="true"></a> | <a href="https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/nanopore-16S-metagenomics/workflows/" class="fas fa-share-alt" aria-hidden="true"></a>
+| All | Shotgun metagenomics | Short reads | Taxonomic Profiling, Functional Analysis | | <a href="https://asaim.readthedocs.io/en/latest/workflows.html"> | ASaiM (missing reference)
```

Comment:
I cannot see this link on the page. Guess it has to like `<a href="" class="fas fa-graduation-cap" aria-hidden="true"></a>` What is ASaiM btw ?